index.html

```diff
@@ -8,7 +8,6 @@
 </head>
 
 <body>
-
```

Comment:
good job almost there  :fist_oncoming:
![Alt Text](https://c.tenor.com/7dH7xotTn6sAAAAM/work-is-almost-over-happy.gif)
-please be carful  with blank lines and indentation only use them for readability purpose

cheers :smile: